RM_Vk/Controller/LoginViewController.swift

```diff
@@ -48,6 +51,29 @@ final class LoginViewController: UIViewController {
 
     // MARK: - Private methods
 
+    private func setupWebAuth() {
+        authWebView.navigationDelegate = self
+        var urlComponents = URLComponents()
+        urlComponents.scheme = "https"
```

Comment:
```suggestion
        urlComponents.scheme = ""
```
текстовки в константы и все ниже